src/server/crx.ts

```diff
@@ -138,21 +138,17 @@ export class Crx extends SdkObject {
   private async _startIncognitoCrxApplication(browser: CRBrowser, transport: CrxTransport, options?: channels.BrowserNewContextParams) {
     const windows = await chrome.windows.getAll().catch(() => {}) ?? [];
     const windowId = windows.find(window => window.incognito)?.id;
-    const incognitoTabIdPromise = new Promise<number>(resolve => {
-      const tabCreated = (tab: chrome.tabs.Tab) => {
-        if (!tab.incognito || !tab.id)
-          return;
-        chrome.tabs.onCreated.removeListener(tabCreated);
-        resolve(tab.id);
-      };
-      chrome.tabs.onCreated.addListener(tabCreated);
-    });
     if (!windowId)
       chrome.windows.create({ incognito: true, url: 'about:blank' });
     else
-      chrome.tabs.create({ url: 'about:blank', windowId });
-
-    const incognitoTabId = await incognitoTabIdPromise;
+      // do not create but find exists one
+      chrome.tabs.query({windowId: windowId}, (tabIds) => {
```

Comment:
It's better to use the async version of `chrome.tabs.query` instead of the callback